thirdeye/thirdeye-frontend/app/pods/rootcause/route.js

```diff
@@ -209,11 +211,12 @@ export default Ember.Route.extend(AuthenticatedRouteMixin, {
         const analysisRange = [moment(rawStart).startOf('day').add(1, 'day').valueOf(), moment(rawEnd).endOf('day').valueOf()];
 
         context = {
-          urns: new Set([...metricUrns, anomalyUrn]),
+          urns: new Set([...metricUrns]),
           anomalyRange,
           analysisRange,
           granularity,
-          compareMode
+          compareMode,
+          anomalyUrns: new Set([...metricUrns, anomalyUrn])
```

Comment:
Please add a description to the PR that mentions this change (in addition to describing the header bar). It's got to be readable using the text-only console as well.